css/main.css

```diff
@@ -417,138 +359,149 @@ a:hover {
   width: 675px;
   background-color: #e9e3ed;
   border-radius: 33px;
-}
+  padding: 2%; }
+  .beginner h3,
+  .hardware h3,
+  .general h3,
+  .research h3 {
+    margin: 15px 0; }
+
+.track-text {
+  margin: 0 10px; }
 
 .flickity-page-dots {
-  top: 105%;
-}
+  top: 105%; }
 
 .flickity-page-dots .dot {
   width: 20px;
   height: 20px;
   opacity: 1;
   background: transparent;
-  border: 2px solid #e9e3ed;
-}
+  border: 2px solid #e9e3ed; }
 
 .flickity-page-dots .dot.is-selected {
-  background: #e9e3ed;
-}
+  background: #e9e3ed; }
 
 .gallery {
-  height: 450px;
-}
+  height: 450px; }
 
 @media (max-width: 991px) {
+  .tracks {
+    margin-bottom: 2%; }
   .gallery {
-    height: 500px;
-  }
-
+    height: 500px; }
   .beginner,
-.hardware,
-.general,
-.research {
-    width: 500px;
-  }
-}
+  .hardware,
+  .general,
+  .research {
+    width: 500px; }
+  .track-text {
+    padding: 0px 20px;
+    font-size: 99%; } }
+
 @media (max-width: 767px) {
   .gallery {
     height: 500px;
     width: 100%;
-    position: relative;
-  }
-
+    position: relative; }
   .beginner,
-.hardware,
-.general,
-.research {
-    margin: 0px 3px;
-  }
-}
+  .hardware,
+  .general,
+  .research {
+    margin: 0px 3px; }
+  .track-text {
+    padding: 0px 20px; }
+  .container p {
+    text-align: center; } }
+
 @media (max-width: 575px) {
   .gallery {
-    height: 510px;
-  }
-
+    height: 510px; }
   .beginner,
-.hardware,
-.general,
-.research {
+  .hardware,
+  .general,
+  .research {
     width: 85%;
-  }
-}
+    margin: 0px 3px; }
+  .track-text {
+    padding: 0px 20px; } }
+
 @media (max-width: 500px) {
+  .tracks {
+    margin-bottom: 0; }
   .gallery {
-    height: 540px;
-  }
-}
+    height: 540px; }
+  .track-text {
+    padding: 0px 20px;
+    font-size: 95%; } }
+
 @media (max-width: 425px) {
+  .tracks {
+    margin-bottom: 0; }
```

Comment:
carousel looks great on desktop, but there's too much space between the bottom of the cards & the dots on tablet/mobile:
<img width="81" alt="Screen Shot 2021-07-28 at 10 24 50 AM" src="https://user-images.githubusercontent.com/62481939/127339838-c009c90a-6984-44da-a835-f0aad30fb8d7.png">
<img width="419" alt="Screen Shot 2021-07-28 at 10 20 00 AM" src="https://user-images.githubusercontent.com/62481939/127339821-65de01e6-edc7-4ff9-a335-c36bfabe6bc9.png">